plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ApplicationSourceListPage.java

```diff
@@ -327,7 +335,28 @@ private IObservableValue createLocalTemplateControls(TabFolder tabContainer,
 				btnBrowseFiles, btnBrowseWorkspaceFiles);
 		return localTemplateFilename;
 	}
-
+	
+	private void refreshWizardAfterLocationChange(String filename) {
```

Comment:
I would have chosen a better method name which reflects more what this is doing. Ex. ```loadApplicationSource```